docs/src/modules/components/Notifications.js

```diff
@@ -64,21 +65,7 @@ export default function Notifications() {
     lastSeen: undefined,
     messages: undefined,
   });
-
-  const messageList = messages
-    ? messages
-        .filter((message) => {
-          if (
-            message.userLanguage &&
-            message.userLanguage !== userLanguage &&
-            message.userLanguage !== navigator.language.substring(0, 2)
-          ) {
-            return false;
-          }
-          return true;
-        })
-        .reverse()
-    : null;
+  const { productId } = React.useContext(PageContext);
```

Comment:
I think we need to match `productCategoryId` too. It would be painful to have a MUI X release notification without.